mitosheet/mitosheet/step_performers/merge.py

```diff
@@ -101,6 +109,10 @@ def transpile(
             )
         ]
     
+    
     @classmethod
     def get_modified_dataframe_indexes(cls, params: Dict[str, Any]) -> Set[int]:
+        destination_sheet_index = get_param(params, 'destination_sheet_index')
+        if destination_sheet_index: # If editing an existing sheet, that is what is changed
```

Comment:
I think you have to do an explicit is not None check here -- otherwise if the merge sheet is the first (which is kinda rare, I guess, but can happen), we run into issues.

Do we need to fix this in pivot as well?

Reply:
Yeah I think we do